syntax = "proto3";

package graph_serialize;

message Edge {
    uint32 from = 1;
    uint32 to = 2;
    double weight = 3;
}

message Graph {
    repeated Edge list_edges = 1;
}